meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -27,12 +27,18 @@ PACKAGECONFIG[ubootenv] = ",,,u-boot-fw-utils u-boot-default-env aktualizr-uboot
 SYSTEMD_PACKAGES += "${PN}-lite"
 SYSTEMD_SERVICE_${PN}-lite = "aktualizr-lite.service aktualizr-lite.path"
 
+COMPOSE_HTTP_TIMEOUT = "60"
+
 # Workaround as aktualizr is a submodule of aktualizr-lite
 do_configure_prepend_lmp() {
     cd ${S}
     git log -1 --format=%h | tr -d '\n' > VERSION
 }
 
+do_compile() {
+    sed -e 's/@@COMPOSE_HTTP_TIMEOUT@@/${COMPOSE_HTTP_TIMEOUT}/' ${WORKDIR}/aktualizr-lite.service.in > ${WORKDIR}/aktualizr-lite.service
```

Comment:
Here you are replacing the do_compile function entirely, use do_compile_append_lmp instead.

Reply:
Yep copy / paste error here.  Thanks again.